.mci.yml

```diff
@@ -333,9 +333,11 @@ functions:
                   export DYLD_LIBRARY_PATH=.:$PREFIX/lib/
                   export PATH=$(pwd)/src/bsoncxx/${build_type}:$(pwd)/src/mongocxx/${build_type}:$PREFIX/bin:$(pwd)/install/bin:$PATH
 
-                  # CRUD_TESTS_PATH and GRIDFS_TESTS_PATH are required to be set by the
-                  # `test_crud_specs` and `test_gridfs_specs` test binaries, respectively.
-                  export CRUD_TESTS_PATH="$(pwd)/../data/crud"
+                  # CRUD_LEGACY_TESTS_PATH, CRUD_UNIFIED_TESTS_PATH and GRIDFS_TESTS_PATH are required to
+                  # be set by the `test_crud_specs`, `test_unified_format_spec` and `test_gridfs_specs` test
+                  # binaries respectively.
```

Comment:
Is this comment necessary? All of the below `PATH` variables are for their respective spec test targets; not sure why `CRUD_TESTS_PATH` and `GRIDFS_TESTS_PATH` needed to be singled out.

Reply:
Removed!